# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_alt-data_simple.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_alt-data_simple.js
--- a/netwerk/test/unit/test_alt-data_simple.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_alt-data_simple.js	Tue Feb 12 16:08:25 2019 +0000
@@ -92,7 +92,7 @@
   var cc = chan.QueryInterface(Ci.nsICacheInfoChannel);
   cc.preferAlternativeDataType(altContentType, "");
 
-  chan.asyncOpen2(new ChannelListener(readServerContent, null));
+  chan.asyncOpen(new ChannelListener(readServerContent, null));
 }
 
 function readServerContent(request, buffer)
@@ -139,7 +139,7 @@
   cc.preferAlternativeDataType(altContentType, "text/plain");
   cc.preferAlternativeDataType("dummy2", "");
 
-  chan.asyncOpen2(new ChannelListener(readAltContent, null));
+  chan.asyncOpen(new ChannelListener(readAltContent, null));
 }
 
 function readAltContent(request, buffer)
@@ -178,7 +178,7 @@
   var chan = make_channel(URL);
   var cc = chan.QueryInterface(Ci.nsICacheInfoChannel);
   cc.preferAlternativeDataType(altContentType, "");
-  chan.asyncOpen2(new ChannelListener(readEmptyAltContent, null));
+  chan.asyncOpen(new ChannelListener(readEmptyAltContent, null));
 }
 
 function readEmptyAltContent(request, buffer)